addons/mrp/static/tests/mrp_widgets.test.js

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { defineModels, fields } from "@web/../tests/web_test_helpers";
+import { mailModels, openFormView, start, startServer } from "@mail/../tests/mail_test_helpers";
+import { DEFAULT_MAIL_VIEW_ID } from "@mail/../tests/mock_server/mock_models/constants";
+
+describe.current.tags("desktop");
+
+class ResFake extends mailModels.ResFake {
+    _views = {
+        [`form,${DEFAULT_MAIL_VIEW_ID}`]: /* xml */ `
```

Comment:
is the `xml` comment really useful here? Also, why don't we use `defineMrpModels`? Is this model extension conflicting with other tests?

Reply:
I think some IDEs support this comment, it should be highlighted as XML instead of a string.
I moved it inside mock_models